back/src/api/albums.rs

```diff
@@ -350,9 +351,40 @@ pub async fn edit_album(
         }
 
         if patch != EMPTY_ALBUM_PATCH {
+            if patch.shared_with.is_some() {
+                let user_ids = users::table
+                    .select(users::id)
+                    .filter(users::email.eq_any(patch.shared_with.as_ref().unwrap()))
+                    .load::<Uuid>(conn)
+                    .map_err(DatabaseError::<ApiError>::from)?;
+
+                diesel::delete(shared_albums::table.filter(shared_albums::album_id.eq(*album_id)))
+                    .execute(conn)
+                    .map_err(DatabaseError::<ApiError>::from)?;
+
+                // Insert the new shared users
+                insert_into(shared_albums::table)
```

Comment:
Be consistent, either use `diesel::delete(), diesel::update(), ...` everywhere ***OR*** use imported functions `delete(), update()`